lib/slime/parser.ex

```diff
@@ -129,15 +138,19 @@ defmodule Slime.Parser do
   end
 
   defp parse_attributes(""), do: {"", []}
-  defp parse_attributes("(" <> line), do: parse_wrapped_attributes(line, ")")
-  defp parse_attributes("[" <> line), do: parse_wrapped_attributes(line, "]")
-  defp parse_attributes("{" <> line), do: parse_wrapped_attributes(line, "}")
+
   defp parse_attributes(line) do
-    match = @attr_group_regex |> Regex.run(line) |> List.first
-    offset = String.length(match)
-    {attrs, rem} = line |> String.split_at(offset)
-    attrs = parse_attributes(attrs, [])
-    {rem, attrs}
+    delim = String.first(line)
+    if Dict.has_key?(@attr_list_delims , delim) do
```

Comment:
May be we should provide better error message for case when closing delim is not what we are expecting, for example `div[id="test"}` will result in match error in `parse_wrapped_attributes` as far as i see.